parsers/ttcn.c

```diff
@@ -894,7 +897,7 @@ static int parseTypeDefBody (void)
 			/* SubTypeDef */
 			ungetToken();
 			/* Stop after type name, no need to parse ArrayDef, StringLength, etc */
-			return (parseType() && parseID(K_TYPE));
+			return parseType() && parseID(K_TYPE);
```

Comment:
Do we need this change for supporting advanced parameterization?
At least this chnage should go to a separated commit.

Reply:
we don't need this, i removed this changed and squashed commits into one.